plugins/src/gcc/imp.rs

```diff
@@ -0,0 +1,1274 @@
+///! Implements https://datatracker.ietf.org/doc/html/draft-ietf-rmcat-gcc-02
```

Comment:
This could use a quick description of the element's design (implements pacing, performs estimation upon receiving downstream events, typically placed in front of `rtpsession`, ..)